docs/pages/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,313 @@
+
+![Click on the service account](../../../img/helm/gcp/10-serviceaccountdetails.png)
+
+2. Click the "Keys" tab at the top and click "Add Key". Choose "JSON" and click "Create".
+
+![Create JSON key](../../../img/helm/gcp/11-createkey.png)
+
+3. The JSON private key will be downloaded to your computer. Take note of the filename (`bens-demos-24150b1a0a7f.json` in this example)
+   as you will need it shortly.
+
+![Private key saved](../../../img/helm/gcp/12-privatekey.png)
+
+#### Create the Kubernetes secret containing the JSON private key for the service account
+
+Find the path where the JSON private key was just saved (likely your browser's default "Downloads" directory).
+
+Use `kubectl` to create the secret, using the path to the JSON private key:
+
+```shell
+kubectl create namespace teleport && \
+kubectl --namespace teleport create secret generic teleport-gcp-credentials --from-file=gcp-credentials.json=/path/to/downloads/bens-demos-24150b1a0a7f.json
+```
+
+<Admonition type="tip">
+  If you installed the Teleport chart into a specific namespace, the `teleport-gcp-credentials` secret you create must also be added to the same namespace.
+</Admonition>
+
+<Admonition type="note">
+ The default name configured for the secret is `teleport-gcp-credentials`.
+
+ If you already have a secret created, you can skip this creation process and specify the name of the secret using `gcp.credentialSecretName`.
+
+ The credentials file stored in any secret used must have the key name `gcp-credentials.json`.
+</Admonition>
+
+### Choose how to configure the cluster
+
+There are two different ways to configure the `teleport-cluster` Helm chart to use `gcp` mode - using a `values.yaml` file or using `--set`
+on the command line.
+
+We recommend using a `values.yaml` file as it can be easily kept in source control.
+
+The `--set` CLI method is more appropriate for quick test deployments.
+
+<Tabs>
+  <TabItem label="Using values.yaml">
+  Create a `gcp-values.yaml` file and write the values you've chosen above to it:
+
+  ```yaml
+  clusterName: teleport.example.com
+  chartMode: gcp
+  gcp:
+    projectId: gcpproj-123456
+    backendTable: teleport-helm-backend
+    auditLogTable: teleport-helm-events
+    sessionRecordingBucket: teleport-helm-sessions
+  highAvailability:
+    replicaCount: 2
+  ```
+
+  Install the chart with the values from your `gcp-values.yaml` file using this command:
+
+  ```shell
+  helm install teleport teleport/teleport-cluster \
+    --create-namespace \
+    --namespace teleport \
+    -f gcp-values.yaml
+  ```
+
+  </TabItem>
+  <TabItem label="Using --set via CLI">
+  Install the chart using this command, replacing the placeholders with the values you've chosen above:
+
+  ```shell
+  helm install teleport teleport/teleport-cluster \
+    --create-namespace \
+    --namespace teleport \
+    --set clusterName=teleport.example.com \
+    --set chartMode=gcp \
```

Comment:
I like the new chart structure with modes and namespaced arguments, looks much better then the old cluster, great job.